include/zcbor_common.h

```diff
@@ -109,39 +113,43 @@ do {\
 
 
 #define ZCBOR_VALUE_IN_HEADER 23 ///! Values below this are encoded directly in the header.
-
 #define ZCBOR_BOOL_TO_PRIM 20 ///! In CBOR, false/true have the values 20/21
 
-#define ZCBOR_FLAG_RESTORE 1UL ///! Restore from the backup.
-#define ZCBOR_FLAG_CONSUME 2UL ///! Consume the backup.
+#define ZCBOR_FLAG_RESTORE 1UL ///! Restore from the backup. Overwrite the current state with the state from the backup.
```

Comment:
Should we keep within 100 char witdth?

Reply:
I think there is a gain in readability this way, so IMO this type of inline docs are a valid exception to the line width rule.